tests/robot/OBSERVATION/01_create.robot

```diff
@@ -87,7 +87,7 @@ Force Tags              observation_create    create
 
 005 Create Sofa Score
 	[Documentation]     1. *CREATE* new EHR record\n\n 
-	...                 2. *LOAD* _create-sofa-score.json_\n\n
+	...                 2. *LOAD* _create-sofa-score-cardiovaskular-score-1.json_\n\n
```

Comment:
Please dont edit the robot tests ;)

Reply:
@af-pwo the documentation was wrong: it says is loading a different file.